contributing/releasing.md

```diff
@@ -231,6 +231,9 @@ synchronization within Google.
 
     git push origin stable develop
 
+You can now sync to the desired stable release. [go/mdc-releasing#re-run-the-import-script-against-githubstable](http://go/mdc-releasing#re-run-the-import-script-against-githubstable). Once you've submitted
+the internal CL, continue below to tag and publish the release.
```

Comment:
I disagree here. We can still tag and publish even if the internal CL is not yet submitted. Internally we can lag behind source of truth by a version or two. (not desirable but possible)

We can adjust the wording here to reflect that possibility by removing the last sentence.

Reply:
I see what you did here now. I still disagree. Sorry for not responding sooner.
My understanding of what these new instruction do:
We are allowed to push to stable without tagging the commit as a release. Deviating from Git-flow. If you need to you can "work on stable"? to make the necessary rollbacks to make the internal sync happen.
We are blocking the tag based on if we have landed the internal sync. We need a new dragon.

Additionally: The internal sync CL explicitly points to the [tagged publish release](http://go/mdc-releasing#update-the-cls-description). We should make our descriptions clear that we are not syncing to a published release but to a candidate.

I would like to counter propose:
Once you pushed to stable you are done with the release. tag and publish. If you made a mistake you can do a hotfix. Internal sync will just happen on that next release.